src/main/java/Duke.java

```diff
@@ -10,38 +10,37 @@ public static void main(String[] args) {
         Scanner stringScanner = new Scanner(System.in);
         String userQuery = stringScanner.nextLine();
         String userQueryReturn = "";
-        //String[] listItems = new String[100];
         ArrayList<Task> taskItems = new ArrayList<Task>();
         int itemCount = 0;
         while(!userQuery.equalsIgnoreCase("bye")){
-            taskItems.add(new Task(userQuery));
             if(userQuery.contains("done")){
                 int doneIndex = Integer.parseInt(userQuery.substring(5));
                 taskItems.get(doneIndex-1).setStatus(true);
                 userQueryReturn = "____________________________________________________________\n"
                         +" Nice! I've marked this task as done: \n"
-                        + " ["+taskItems.get(doneIndex-1).getStatusIcon()+"] "+taskItems.get(doneIndex-1).getDescription()+"\n"
+                        + " ["+taskItems.get(doneIndex-1).getTypeIcon()+"]["+taskItems.get(doneIndex-1).getStatusIcon()+"] "+taskItems.get(doneIndex-1).getDescription()+"\n"
                         + "____________________________________________________________\n";
                 System.out.println(userQueryReturn);
                 userQuery = stringScanner.nextLine();
             }else if(!userQuery.equalsIgnoreCase("list")) {
```

Comment:
Hello, functionality looks good! However, I suggest formatting into more methods such as checkDone so you code looks cleaner